bundles/org.openhab.binding.iaqualink/src/main/java/org/openhab/binding/iaqualink/internal/IAqualinkBindingConstants.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.iaqualink.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.eclipse.smarthome.core.thing.type.ChannelTypeUID;
+
+/**
+ * The {@link IAqualinkBindingConstants} class defines common constants, which are used
+ * across the whole binding.
+ *
+ * @author Dan Cunningham - Initial contribution
+ */
+@NonNullByDefault
+public class IAqualinkBindingConstants {
+
+    public static final String BINDING_ID = "iaqualink";
+    public static final String CHANNEL_TYPE_AUX_SWITCH = "aux-switch";
+    public static final String CHANNEL_TYPE_AUX_DIMMER = "aux-dimmer";
+    public static final String CHANNEL_TYPE_AUX_JANDYCOLOR = "aux-jandycolor";
+    public static final String CHANNEL_TYPE_AUX_PENTAIRSAM = "aux-pentairsam";
+    public static final String CHANNEL_TYPE_AUX_JANDYLED = "aux-jandyled";
+    public static final String CHANNEL_TYPE_AUX_PENTAIRIB = "aux-pentairib";
+    public static final String CHANNEL_TYPE_AUX_HAYWARD = "aux-hayward";
+
+    public static final String CHANNEL_TYPE_ONETOUCH = "onetouch";
+
+    public static final ThingTypeUID IAQUALINK_DEVICE_THING_TYPE_UID = new ThingTypeUID(BINDING_ID, "controller");
+
+    public static final ChannelTypeUID CHANNEL_TYPE_UID_ONETOUCH = new ChannelTypeUID(
+            BINDING_ID + ":" + CHANNEL_TYPE_ONETOUCH);
```

Comment:
Any reason not to use create ChannelTypeUID using separate arguments, like with the other constants that use binding id?
```suggestion
            BINDING_ID, CHANNEL_TYPE_ONETOUCH);
```